index.js

```diff
@@ -1,5 +1,5 @@
 function constructRequest(har, userAgent = false) {
-  if (!har) throw new Error('Missing har file');
+  if (!har) throw new Error('Missing HAR file');
```

Comment:
Should this be a major version bump if we're changing error messages?

Reply:
Yeah it's going to be a major version bump anyways since it's changing how it's handling the `postData` object.